diagram-viewer/src/main/java/com/powsybl/diagram/viewer/DiagramViewer.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.diagram.viewer;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.image.Image;
+import javafx.scene.layout.BorderPane;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * @author Florian Dupuy <florian.dupuy at rte-france.com>
+ */
+public class DiagramViewer extends Application {
+
+    public void start(Stage primaryStage) throws IOException {
+        FXMLLoader loader = new FXMLLoader();
+        loader.setLocation(Objects.requireNonNull(getClass().getResource("/mainView.fxml")));
+        BorderPane root = loader.load();
+        primaryStage.setTitle("Developer viewer for powsybl-diagram");
+        // Logo not displayed for ubuntu users
+        // see https://bugs.openjdk.org/browse/JDK-8092837
```

Comment:
As mentioned you can remove this comment, the bugs are not marked as resolved but I've always seen the logo on Ubuntu since the logo has been added to the viewers.